### ISSUE TITLE
timetable in landscape difficult to follow

### ISSUE THREAD

**Issue by nutpantz** [TylerWilliamson/QuickWeather#97]
![Screenshot_20231005_233415_QuickWeather](https://github.com/TylerWilliamson/QuickWeather/assets/12160507/db33eada-cbd7-4b56-bc3a-d29931807fba)

It's just hard to follow how the times are shown

**Comment by TylerWilliamson** [comment]
Landscape mode has been a challenge for me, and it is a lower-priority item, but I will keep this issue open to help track the progress.

As for this specific issue, it's because the cards fill in left-to-right, top-to-bottom. I can adjust this though.

**Comment by DJCrashdummy** [comment]
well, it's not just the timetable which can get messy... all landscape views as a whole are kind of "unfortunate" to say at least. :grimacing:
![Screenshot1](https://github.com/user-attachments/assets/f8672aa5-6d46-4f3e-8301-0b4af775d623)
![Screenshot2](https://github.com/user-attachments/assets/7ca821c3-1dff-48ce-a17e-efdfc78a848d)
